frontend/src/data/variables/BrfssProvider.ts

```diff
@@ -98,8 +96,26 @@ class BrfssProvider extends VariableProvider {
           estimated_total_diabetes: (series) => series.sum(),
         })
         .resetIndex();
+
+      // We need to get the national acs dataset here in order to
+      // get the national pct share of population for each state.
+      acsBreakdowns.geography = "national";
+      const acsQueryResponse = await this.acsProvider.getData(
```

Comment:
Can we name acsQueryResponse here to distinguish it from that on line 55? acsNationalQueryResponse and acsStateQueryResponse?